app/helpers/application_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ApplicationHelper
+    def full_title(page_title = "")

```

Comment:
1. bỏ ngoặc được nhé 
2. `= ""` xóa dc ko cu 
3. `"Ruby on Rails Tutorial Sample App"` tìm hiểu và sử dụng I18n nhé em
tất cả các text mà hiển thị lên view của người dùng, toàn bộ đều phải dùng i18n nhé
4. Phân biệt blick?, empty? nhé
trả lời ở đây

Reply:
2. = "" có thể xóa nhưng nó là giá trị mặc định của function nếu xóa thì khi gọi function ta phải truyền parameter vào
4. Em chỉ biết .empty? sẽ kiểm tra chuỗi đó có rỗng hay không ? Chấp nhận cả chuỗi có chứa khoảng trắng. Còn .blick? thì em không biết ạ

Reply:
`Còn .blick? thì em không biết ạ em research nhưng không thấy function này ạ`
>> chị ghi sai, `blank?` á :v 

Cơ mà chỗ empty, em còn thiếu là loại dữ liệu là collection mới gọi empty? dc